lib/browser/init.js

```diff
@@ -163,7 +164,7 @@ require('./api/protocol')
 const mainStartupScript = packageJson.main || 'index.js'
 
 // Workaround for electron/electron#5050 and electron/electron#9046
-if (process.platform === 'linux' && ['Pantheon', 'Unity:Unity7', 'ubuntu:GNOME'].includes(process.env.XDG_CURRENT_DESKTOP)) {
+if (process.platform === 'linux' && ['Pantheon', 'Unity:Unity7', 'ubuntu:GNOME', 'pop:GNOME'].includes(process.env.XDG_CURRENT_DESKTOP)) {
```

Comment:
is this really relevant to the integrity verification problem? which problem does this solve?

Reply:
It is not relevant for the integrity verification.

(my changes were previously made on Master-Branch of electron)